experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.Handoff/TurnContextExtender.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Handoff
+{
+    /// <summary>
+    /// Extends ITurnContext to add InitiateHandoffAsync.
+    /// </summary>
+    public static class TurnContextExtender
```

Comment:
Naming, we have a ITurnContextExtensions could this be a HandoffExtensions?

Reply:
Done